src/components/chat/comment-card.tsx

```diff
@@ -0,0 +1,100 @@
+import { inject, observer } from "mobx-react";
+import React from "react";
+import { BaseComponent } from "../base";
+import DocumentCommentIcon from "../../assets/document-id.svg";
+import SendIcon from "../../assets/send-icon.svg";
+import "./comment-card.scss";
+import "../themes.scss";
+
+interface IProps {
+}
+
+interface IState {
+  commentTextAreaHeight: number | string;
+  commentAdded: boolean;
+  commentText: string;
+}
+
+@inject("stores")
+@observer
+export class CommentCard extends BaseComponent<IProps, IState> {
```

Comment:
I'm fine with leaving this as a class component but it could have been written as a functional component as well.